Builds/CMake/RippledCov.cmake

```diff
@@ -2,26 +2,10 @@
    coverage report target
 #]===================================================================]
 
-ProcessorCount(PROCESSOR_COUNT)
-
 if (coverage)
```

Comment:
We don't even include this file unless `coverage=ON` right? Can we just assert that `coverage=ON` here and de-dent the rest of the file?

Reply:
will do, thanks